src/odemis/util/spot.py

```diff
@@ -255,17 +256,19 @@ def MaximaFind(image, qty, len_object=18):
 
     # dilation
     structure = _CreateSEDisk(len_object // 2)
-    dilated = cv2.dilate(filtered, structure, iterations=1)
+    dilated = cv2.dilate(filtered, structure)
```

Comment:
Why did you remove this, is `iterations=1` the default?

Reply:
Yes; default is iterations=1; see [link](https://docs.opencv.org/2.4/modules/imgproc/doc/filtering.html#void%20dilate(InputArray%20src,%20OutputArray%20dst,%20InputArray%20kernel,%20Point%20anchor,%20int%20iterations,%20int%20borderType,%20const%20Scalar&%20borderValue))